lbry/lbry/extras/daemon/Daemon.py

```diff
@@ -3270,6 +3270,374 @@ def jsonrpc_stream_cost_estimate(self, uri):
+
+                                                          ... --locations="42.990605:-71.460989"
+
+                                                        finally, you can also pass JSON string of dictionary
+                                                        on the command line as you would via JSON RPC
+
+                                                          ... --locations="{'country': 'US', 'state': 'NH'}"
+
+            --clear_locations              : (bool) clear existing locations (prior to adding new ones)
+            --thumbnail_url=<thumbnail_url>: (str) thumbnail url
+            #--cover_url=<cover_url>        : (str) url of cover image
+            --account_id=<account_id>      : (str) account in which to look for collection (default: all)
+            --wallet_id=<wallet_id>        : (str) restrict operation to specific wallet
+          --funding_account_ids=<funding_account_ids>: (list) ids of accounts to fund this transaction
+            --claim_address=<claim_address>: (str) address where the collection is sent
+            --new_signing_key              : (bool) generate a new signing key, will invalidate all previous publishes
+            --preview                      : (bool) do not broadcast the transaction
+            --blocking                     : (bool) wait until transaction is in mempool
+            --replace                      : (bool) instead of modifying specific values on
+                                                    the collection, this will clear all existing values
+                                                    and only save passed in values, useful for form
+                                                    submissions where all values are always set
+
+        Returns: {Transaction}
+        """
+        wallet = self.wallet_manager.get_wallet_or_default(wallet_id)
+        funding_accounts = wallet.get_accounts_or_all(funding_account_ids)
+        if account_id:
+            account = wallet.get_account_or_error(account_id)
+            accounts = [account]
+        else:
+            account = wallet.default_account
+            accounts = wallet.accounts
+
+        existing_collections = await self.ledger.get_collections(
+            wallet=wallet, accounts=accounts, claim_id=claim_id
+        )
+        if len(existing_collections) != 1:
+            account_ids = ', '.join(f"'{account.id}'" for account in accounts)
+            raise Exception(
+                f"Can't find the collection '{claim_id}' in account(s) {account_ids}."
+            )
+        # Here we might have a problem of replacing a stream with a collection
+        old_txo = existing_collections[0]
+        if not old_txo.claim.is_collection:  # as we're only checking @ or not, this is not definitive
+            raise Exception(
+                f"A claim with id '{claim_id}' was found but it is not a stream or collection."
```

Comment:
You're checking if it's a `is_collection` so it might actually be a stream but not a collection, message says `stream or colleciton`, should just be `collection`.